recipes/openpmix/meta.yaml

```diff
@@ -0,0 +1,131 @@
+  # rename on feedstock
+  name: libpmix
+  version: {{ version }}
+
+source:
+  url: https://github.com/openpmix/openpmix/releases/download/v{{ version }}/pmix-{{ version }}.tar.bz2
+  sha256: f72d50a5ae9315751684ade8a8e9ac141ae5dd64a8652d594b9bee3531a91376
+
+build:
+  skip: true  # [win]
+  number: 0
+
+# one top-level build
+# outputs select which files to include
+# from this build
+# but don't run any build scripts themselves
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - make
+    - pkg-config
+    - python 3.12
+  host:
+    - libevent
+    - libhwloc
+    - zlib
+
+outputs:
+  - name: libpmix
+    files:
+      # libpmix has everything except for specific exclusions
+      include:
+        - '**/*'
+      exclude:
+        # these are in -bin
+        - bin/
+        # these are in -devel
+        - include/
+        - lib/pkgconfig/
+        # exclude docs (not packaged)
+        - share/doc/
+        - share/man/
+    build:
+      run_exports:
+        - {{ pin_subpackage('libpmix', max_pin='x.x') }}
+    requirements:
+      # only need requirements with run_exports here
+      # no build is actually run
+      # TODO: remove extras when recipe name is fixed to not match this output
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - make
+        - pkg-config
+        - python 3.12
+      host:
+        - libevent
+        - libhwloc
+        - zlib
+
+    test:
+      requires:
+        - pkg-config
+      commands:
+        - test -f $PREFIX/lib/libpmix${SHLIB_EXT}
+        - test ! -f $PREFIX/lib/libpmpix.a
+        - test ! -f $PREFIX/bin/pmix_info
+        - test ! -d $PREFIX/include/pmix/src
+        - test ! -d $PREFIX/share/doc/pmix
+        - pkg-config --cflags --libs pmix
+
+  - name: libpmix-devel
+    build:
+      run_exports:
+        - {{ pin_subpackage('libpmix', max_pin='x.x') }}
+    files:
+      include:
+        - include/
+        - lib/pkgconfig/
+    requirements:
+      run:
+        - {{ pin_subpackage('libpmix', exact=True) }}
+    test:
+      requires:
+        - pkg-config
+      commands:
+        - test -f $PREFIX/include/pmix.h
+        - test -d $PREFIX/include/pmix/src
+        - test -f $PREFIX/lib/libpmix${SHLIB_EXT}
+        - pkg-config --cflags --libs pmix
+
+  - name: pmix-bin
+    files:
+      # should only include files excluded by libpmix
+      include:
+        - bin/
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+      host:
+        - {{ pin_subpackage('libpmix', exact=True) }}
+      run:
+        - {{ pin_subpackage('libpmix', exact=True) }}
+    test:
+      commands:
+        - pmix_info
+
+about:
+  home: https://pmix.org
+  summary: OpenPMIx Implementation of the PMIx Standard
+  description: |
+    Reference Implementation of the Process Management Interface Exascale (PMIx) standard
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
The `license_family` key is now redundant and can be removed because the `license` key is already linted to be a valid SPDX identifier.